src/js/components/Breadcrumb.js

```diff
@@ -242,18 +232,23 @@ class Breadcrumb extends React.Component {
       // Decode Label
       let label = decodeURIComponent(pathParts[pathParts.length - 1]);
       let labelIsParam = label.charAt(0) === ':' && label.slice(1) in params;
+      if (labelIsParam) {
+        // Remove colon
+        label = label.slice(1);
+      }
 
       let crumbObjects = buildCrumb(
         label,
         name,
         params,
-        labelIsParam
+        labelIsParam,
+        buildCrumbCustomParams
```

Comment:
I think we should put this argument in the `buildCrumbs` function at the top and we should use it in that default `buildCrumbs` method.